vllm/sequence.py

```diff
@@ -524,6 +527,9 @@ def get_seqs(
             seq for seq in self.seqs_dict.values() if seq.status == status
         ]
 
+    def get_encoder_seq(self) -> Sequence:
+        return self.encoder_seq  # type: ignore
```

Comment:
Should the return type here be `Optional[Sequence]` so that we don't need a `# type: ignore` and callers know that they have to check for None?

Reply:
Thanks, committed this change.